build/package/Installer.RPM.targets

```diff
@@ -0,0 +1,213 @@
+      <ConfigJsonFile>$(RepoRoot)/packaging/rpm/dotnet-config.json</ConfigJsonFile>
+      <RpmIntermediatesDir>$(IntermediateDirectory)/$(RpmPackageName)/$(RpmPackageVersion)</RpmIntermediatesDir>
+      <RpmTestResultsXmlFile>$(RpmIntermediatesDir)/debian-testResults.xml</RpmTestResultsXmlFile>
+      <RpmInstalledDirectory>/usr/share/dotnet</RpmInstalledDirectory>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <rpmLayoutDirectory>$(RpmIntermediatesDir)/rpmLayoutDirectory/</rpmLayoutDirectory>
+      <rpmLayoutPackageRoot>$(rpmLayoutDirectory)package_root</rpmLayoutPackageRoot>
+      <rpmLayoutSDK>$(rpmLayoutPackageRoot)/sdk</rpmLayoutSDK>
+      <rpmLayoutDocs>$(rpmLayoutDirectory)docs</rpmLayoutDocs>                <!-- Man Pages -->
+      <rpmLayoutTemplates>$(rpmLayoutDirectory)templates</rpmLayoutTemplates> <!-- Copyright, Changelog -->
+      <rpmLayoutScripts>$(rpmLayoutDirectory)scripts</rpmLayoutScripts>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <SdkRpmPackageVersion>$(SdkVersion)</SdkRpmPackageVersion>
+      <SdkRpmPackageName>$(ArtifactNameCombinedHostHostFxrFrameworkSdk)-$(SdkRpmPackageVersion)</SdkRpmPackageName>
+      <SharedFxRpmPackageVersion>$(SharedFrameworkVersion)</SharedFxRpmPackageVersion>
+      <SharedFxRpmPackageName>dotnet-runtime-$(SharedFxRpmPackageVersion)</SharedFxRpmPackageName>
+      <SharedFxRpmPackageName>$(SharedFxRpmPackageName.ToLower())</SharedFxRpmPackageName>
+      <HostFxrRpmPackageVersion>$(HostFxrVersion)</HostFxrRpmPackageVersion>
+      <HostFxrRpmPackageName>dotnet-hostfxr-$(HostFxrRpmPackageVersion)</HostFxrRpmPackageName>
+      <HostFxrRpmPackageName>$(HostFxrRpmPackageName.ToLower())</HostFxrRpmPackageName>
+      <HostRpmPackageName>dotnet-host</HostRpmPackageName>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <EndToEndTestProject>$(RepoRoot)/test/EndToEnd/EndToEnd.csproj</EndToEndTestProject>
+    </PropertyGroup>
+
+    <RemoveDir Condition="Exists('$(RpmIntermediatesDir)')" Directories="$(RpmIntermediatesDir)"/>
+    <MakeDir Directories="$(RpmIntermediatesDir)"/>
+
+    <!-- Create empty rpm layout -->
+    <RemoveDir Condition="Exists('$(rpmLayoutDirectory)')" Directories="$(rpmLayoutDirectory)"/>
+    <MakeDir Directories="$(rpmLayoutDirectory)"/>
+    <MakeDir Directories="$(rpmLayoutPackageRoot)"/>
+    <MakeDir Directories="$(rpmLayoutSDK)"/>
+    <MakeDir Directories="$(rpmLayoutDocs)"/>
+    <MakeDir Directories="$(rpmLayoutTemplates)"/>
+    <MakeDir Directories="$(rpmLayoutScripts)"/>
+
+    <!-- Copy files to rpm layout -->
+    <ItemGroup>
+      <SDKFiles Include="$(InputRoot)/**/*"/>
+      <SDKManpages Include="$(ManPagesDir)/**/*"/>
+      <SDKTemplatesFiles Include="$(TemplatesDir)/**/*"/>
+      <SDKScriptsFiles Include="$(ScriptsDir)/**/*"/>
+      <AspNetRuntimeFilesInput Include="$(AspNetRuntimePackageStorePublishDirectory)/**/*" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(SDKFiles)"
```

Comment:
You can probably reduce this to a single Copy by creating a single "ToCopy" item and adding metadata to it indicating the Destination folder.

You can do it by doing something like:
```
<ItemGroup>
  <SomeGoodName Include="Include="$(InputRoot)/**/*">
    <DestinationFolder>$(rpmLayoutSDK)</DestinationFolder>
  </SomeGoodName>
</ItemGroup>
```

And then, when copying it, instead of having to set a different base to each one, you just access that metadata as the base folder.